shared/chat/manage-channels/delete-channel.js

```diff
@@ -1,8 +1,7 @@
 // @flow
```

Comment:
This screen is inaccessible on mobile. Should be refactored when we allow deleting channels on mobile